spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -428,7 +428,8 @@
             wait_until = nil
 
             freeze_time do
-              wait_until = Time.zone.now + 4.hours
+              wait_until = Time.zone.now +
+                           IdentityConfig.store.in_person_results_delay_in_hours.hours
```

Comment:
This looks fine to me to get the tests passing but I don't understand why it started failing. And I think this version will start failing once we go back to daylight savings in Fall

Reply:
Thank you, yeah, good point. I dug a bit and it seems like it's due to places where the tests use `ActiveSupport::TimeZone['Central Time (US & Canada)']`, which will change offsets across DST, instead of `ActiveSupport::TimeZone[-6], which is what the job itself uses [here](https://github.com/18F/identity-idp/blob/a900e42a7b94a739a71fe234fee64969a05334a0/app/jobs/get_usps_proofing_results_job.rb#L433-L436).